srcpkgs/ibus/template

```diff
@@ -1,28 +1,28 @@
 # Template file for 'ibus'
 pkgname=ibus
-version=1.5.23
+version=1.5.24
 revision=1
 build_style=gnu-configure
 build_helper="gir"
-configure_args="--enable-ui --enable-gtk3 --disable-tests
+configure_args="--enable-ui --enable-gtk3 --enable-gtk4 --disable-tests
  --disable-schemas-compile --enable-memconf --enable-dconf
- --enable-wayland --enable-python2 --with-python=/usr/bin/python3
+ --enable-wayland --disable-python2 --with-python=/usr/bin/python3
  $(vopt_enable dicts emoji-dict) $(vopt_enable dicts unicode-dict)
  --enable-introspection --enable-vala $(vopt_enable ibus_setup setup)"
 hostmakedepends="automake gettext-devel libtool pkg-config intltool dconf
- python3 python glib-devel $(vopt_if vala vala) python-gobject-devel
+ python3 glib-devel $(vopt_if vala vala) python3-gobject-devel
  $(vopt_if dicts 'cldr-emoji-annotation unicode-character-database unicode-emoji')"
 makedepends="dconf-devel gtk+-devel hicolor-icon-theme iso-codes
  json-glib-devel libnotify-devel librsvg-devel python3-xdg
- $(vopt_if vala vala) libXtst-devel"
+ $(vopt_if vala vala) libXtst-devel gtk4-devel"
```

Comment:
Bold move, enabling gtk4 already :D 

What's WIP?

Reply:
Ah, that you haven't tested. Will ask for reviewers.

Reply:
Yes I couldn't test it.